spec/javascripts/NewService/NewServiceForm.spec.jsx

```diff
@@ -61,28 +60,6 @@ it('should render the correct form depending on which mode is selected', () => {
   expect(wrapper.find('ServiceDiscoveryForm').exists()).toEqual(false)
 })
 
-describe('when Api as Product is enabled', () => {
-  beforeAll(() => {
-    props.apiap = true
-  })
-
-  it('should render itself', () => {
-    const wrapper = mount(<NewServiceForm {...props}/>)
-    expect(wrapper.find('#new_service_source').exists()).toEqual(true)
-    expect(wrapper.find(`input[name='source']`).length).toEqual(2)
-
-    const clickEvent = value => ({ currentTarget: { value } })
-
-    act(() => {
-      wrapper.find('input#source_manual').props().onChange(clickEvent('manual'))
-    })
-
-    wrapper.update()
-    expect(wrapper.find('h1').text()).toEqual('New Product')
-    expect(wrapper.find('.pf-c-button.create').text()).toEqual('Create Product')
-  })
```

Comment:
The test is still valid, only the `describe` and `beforeAll` are outdated.

Reply:
You are right